docs/reference/dqm.md

```diff
@@ -0,0 +1,79 @@
+# Data Quality Monitoring
+
+Data Quality Monitoring (DQM) is a Feast module aimed to help users to validate their data with the user-curated set of rules.
+Validation could be applied during:
+* Historical retrieval (training dataset generation)
+* [planned] Writing features into an online store
+* [planned] Reading features from an online store
+
+Its goal is to address several complex data problems, namely:
+* Data Consistency - new training dataset could be significantly different from the previous, which might require a change in model architecture.
+* Issues/bugs in the upstream pipeline - bug in upstream could cause invalid values to overwrite existing valid values in an online store.
+* Training/serving skew - distribution shift could significantly decrease the performance of the model.
+
+> By “monitoring data quality” we understand verifying that the characteristics of the tested dataset (we call this characteristics dataset's profile) are "equivalent" to the characteristics of the reference dataset.
+> Eg, data currently passed to the model hasn’t changed significantly since the model was trained, and expectations implicitly made by ML algorithm during training are still met.
```

Comment:
This line is kind of confusing, and I think it's already explained by the previous bullets. Would remove